app/views/devise/mailer/confirmation_instructions.html.slim

```diff
@@ -1,4 +1,4 @@
-p.lead= t('mailer.confirmation_instructions.header', intro: @first_sentence)
+p.lead == t('mailer.confirmation_instructions.header', intro: @first_sentence)
```

Comment:
I think it is more clear to have the keys be explicit that they contain html. I'd say I'm like a 7/10 in terms of how strong I feel about it, but I don't think it's required in order to merge this PR